pursuing-toxic-agenda.html

```diff
@@ -868,7 +868,7 @@ <h2 id="h.va0wuax2pbgz">Environmental Data Justice</h2>
             <p><a href="#ftnt_ref118" id="ftnt118">[118]</a> Kennedy, Miranda. &ldquo;EPA&rsquo;s Chief Environmental Justice Official Steps Down.&rdquo; NPR.Org. Washington, D.C.: National Public Radio, March 13, 2017, <a href="http://www.npr.org/2017/03/13/519954289/epas-chief-environmental-justice-official-steps-down">http://www.npr.org/2017/03/13/519954289/epas-chief-environmental-justice-official-steps-down</a>. Also read Ali&rsquo;s public letter of resignation, accessible at <a href="https://www.documentcloud.org/documents/3514958-Final-Resignation-Letter-for-Administrator.html">https://www.documentcloud.org/documents/3514958-Final-Resignation-Letter-for-Administrator.</a><a href="https://www.documentcloud.org/documents/3514958-Final-Resignation-Letter-for-Administrator.html">html</a><sup><a href="#cmnt2" id="cmnt_ref2">[b]</a></sup></p>
         </div>
         <div>
-            <p><a href="#ftnt_ref119" id="ftnt119">[119]</a> Dillon, <a href="#id.qskbsdioqlsb">interview with Mustafa </a><a href="#id.qskbsdioqlsb">Ali</a>, August 8, 2017; also see Tony Lo Presti,&rdquo; Realizing the Promise of Environmental Civil Rights: The Renewed Effort to Enforce Title VI of the Civil Rights Act of 1964,&rdquo; Administrative Law Review 59, (June 28, 2013): <a href="https://ssrn.com/abstract%3D2286529">https://ssrn.com/abstract=2286529</a>.</p>
+            <p><a href="#ftnt_ref119" id="ftnt119">[119]</a> Dillon, <a href="/mustafa-ali-interview/">interview with Mustafa </a><a href="#id.qskbsdioqlsb">Ali</a>, August 8, 2017; also see Tony Lo Presti,&rdquo; Realizing the Promise of Environmental Civil Rights: The Renewed Effort to Enforce Title VI of the Civil Rights Act of 1964,&rdquo; Administrative Law Review 59, (June 28, 2013): <a href="https://ssrn.com/abstract%3D2286529">https://ssrn.com/abstract=2286529</a>.</p>
```

Comment:
Can we change this to:

`<a href="/mustafa-ali-interview/">interview with Mustafa Ali</a>`